openapi2conv/openapi2_conv_test.go

```diff
@@ -1,11 +1,10 @@
-package openapi2conv_test
+package openapi2conv
```

Comment:
Why the test package rename?

Reply:
avoids needing to import the real package, no need to put the tests in a different package name

Reply:
This makes sure no test code lingers in the actual package. Please revert.